### PR TITLE
[HER-69] Create graph input

### PR DESCRIPTION
**Pull request by Urzica-Platon-Casian**
## Description
Create input type in presentation so it can be used by mutation

## Related
[HER-69](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?issueParent=10022&selectedIssue=HER-69)

### CODE REVIEW

File: src/Presentation/Detection/Common/Types/TCreateDetection.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public sealed class TCreateDetection : ObjectGraphType<MCreateDetection>
+    {
+        public TCreateDetection()
+        {
+            Field(detection => detection.Score).Description("This is the Score");
+            Field(detection => detection.Timestamp).Description("This is the Timestamp");
+            Field(detection => detection.Class).Description("This is the Class");
+        }
+    }
+}
```
Comment (RaduBuciuceanu) [inline src/Presentation/Detection/Common/Types/TCreateDetection.cs:18]: As far as this type will be used for `creation mutation` only, should we consider moving it under the `Detection/Creating/Types` folder?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

File: src/Presentation/Detection/Common/Types/TCreateDetection.cs (hunk 1 above)
Comment (Urzica-Platon-Casian) [reply src/Presentation/Detection/Common/Types/TCreateDetection.cs]: Done

**Review by danidoff — DISMISSED** [review]

File: src/Presentation/Detection/Creating/Types/TCreateDetection.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+namespace Presentation.Detection.Common.Types
```
Comment (RaduBuciuceanu) [inline src/Presentation/Detection/Creating/Types/TCreateDetection.cs:1]: Should we also update the namespace to match the new folder structure.

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

File: src/Presentation/Detection/Creating/Types/TCreateDetection.cs (hunk 2 above)
Comment (Urzica-Platon-Casian) [reply src/Presentation/Detection/Creating/Types/TCreateDetection.cs]: Done

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**